mappings/net/minecraft/client/texture/NativeImage.mapping

```diff
@@ -198,6 +198,8 @@ CLASS net/minecraft/class_1011 net/minecraft/client/texture/NativeImage
 	METHOD method_4327 loadFromTextureImage (IZ)V
 		ARG 1 level
 		ARG 2 removeAlpha
+	METHOD method_47594 copyRect (Lnet/minecraft/class_1011;IIIIIIZZ)V
+		ARG 1 image
```

Comment:
This takes multiple arguments, could you map all?

Reply:
Done!